pkgs/tools/misc/pandoc-include/default.nix

```
@@ -0,0 +1,30 @@
+{ buildPythonApplication
+, fetchFromGitHub
+, lib
+, natsort
+, panflute
+}:
+
+buildPythonApplication rec {
+  pname = "pandoc-include";
+  version = "1.2.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "DCsunset";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-kuxud7m+sWcNqE8A+Fwb8ATgiUwxQvHeYBTyw1UzX4U=";
+  };
+
+  propagatedBuildInputs = [ natsort panflute ];
+
+  doCheck = false;
```

If the package has no check phase you don't need to disable it: Nix will simply skip it.
The repository does have tests, but need pandoc and latex which seems a bit excessive.
As a lightweight alternative you can do:
```suggestion
  pythonImportsCheck = [ "pandoc_include.main" ];
```